client/src/components/FeedbackForm/FeedbackForm.js

```diff
@@ -30,7 +30,7 @@ const FeedbackForm = () => {
 
   const defaultValues = useMemo(
     () => ({
-      subject: 'Feedback for the Frontera Portal',
+      subject: 'Feedback for the Portal',
```

Comment:
Do we need a follow on ticket for Jira to make these things customizable to specific portals (like `subject = "Feedback for the { portalNameFromBackEnd }"` )

Reply:
Ticket created: https://jira.tacc.utexas.edu/browse/FP-883